catalog/src/main/java/com/google/android/fhir/catalog/DemoQuestionnaireViewModel.kt

```diff
@@ -22,27 +22,49 @@ import androidx.lifecycle.SavedStateHandle
 import androidx.lifecycle.viewModelScope
 import ca.uhn.fhir.context.FhirContext
 import ca.uhn.fhir.context.FhirVersionEnum
+import com.google.android.fhir.catalog.DemoQuestionnaireFragment.Companion.QUESTIONNAIRE_ERROR_FILE_PATH_KEY
+import com.google.android.fhir.catalog.DemoQuestionnaireFragment.Companion.QUESTIONNAIRE_FILE_PATH_KEY
+import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
 import org.hl7.fhir.r4.model.QuestionnaireResponse
 
 class DemoQuestionnaireViewModel(application: Application, private val state: SavedStateHandle) :
   AndroidViewModel(application) {
   private val backgroundContext = viewModelScope.coroutineContext
   private var questionnaireJson: String? = null
+  private var questionnaireErrorJson: String? = null
+
+  init {
+    viewModelScope.launch {
+      getQuestionnaireJson()
+      // remove check once all files are added
```

Comment:
```suggestion
      // TODO: remove check once all files are added
```
IDE will highlight it.